### PR TITLE
Feature/numeric stepper props

### PR DESCRIPTION
**Pull request by regis-samurai**
**What problem is this solving?**
This allow to use the unitMultiplier and measurementUnit fields in the product quantity

Most markets need this feature to sell in different units

**How should this be manually tested?**
Link this code version and use it in a store that has different multipliers.
I made another pull request that complement this. In the Numeric Stepper component on vtex.styleguide:  https://github.com/vtex/styleguide/pull/1173

**Screenshots or example usage:**
Captura de Tela 2020-04-20 às 22 51 07
Captura de Tela 2020-04-20 às 21 03 18

### CODE REVIEW

File: react/components/BaseProductQuantity.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { useCallback } from 'react'
 import { NumericStepper } from 'vtex.styleguide'
 import { FormattedMessage } from 'react-intl'
-import { pathOr } from 'ramda'
+import { pathOr, path } from 'ramda'
```
Comment (victorhmp) [inline react/components/BaseProductQuantity.tsx:4]:
```suggestion
import { pathOr } from 'ramda'
```

File: docs/README.md [hunk 2]
```diff
@@ -53,6 +53,7 @@ Check an example of a Product Details Page built using Flex Layout with the `pro
 | `warningQuantityThreshold` | `Number` | Displays the quantity of remaining items in stock if the available quantity is less than or equal to the value given to this property. Default: 0 (does not appear). | `0` |
 | `size` | `NumericSize`| Preset values `font-size` and `padding` to the component | `'small'` |
 | `showLabel` | `boolean` | If it should show a label | `true` |
+| `showMeasurementUnit` | `boolean` | If it should show a measumentUnit | `true` |
```
Comment (victorhmp) [inline docs/README.md:56]:
```suggestion
| `showMeasurementUnit` | `boolean` | If it should show a measumentUnit | `false` |
```

File: docs/README.md [hunk 3]
```diff
@@ -53,6 +53,7 @@ Check an example of a Product Details Page built using Flex Layout with the `pro
 | `warningQuantityThreshold` | `Number` | Displays the quantity of remaining items in stock if the available quantity is less than or equal to the value given to this property. Default: 0 (does not appear). | `0` |
 | `size` | `NumericSize`| Preset values `font-size` and `padding` to the component | `'small'` |
 | `showLabel` | `boolean` | If it should show a label | `true` |
+| `showMeasurementUnit` | `boolean` | If it should show a measumentUnit | `false` |
```
Comment (lbebber) [inline docs/README.md:56]: Props  `unitMultiplier` and `measurementUnit` are missing here

Comment (lbebber) [reply docs/README.md:56]: furthermore, is there any case where `measurementUnit` is present but `showMeasurementUnit` is false? Perhaps the unit should just be always displayed when `measurementUnit` is defined, and you could remove the `showMeasurementUnit` prop

Comment (lbebber) [reply docs/README.md:56]: (nevermind these comments, the changelog was misleading)

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- `unitMultiplier`, `measurementUnit`, and `showMeasurementUnit` props.
```
Comment (lbebber) [inline CHANGELOG.md:11]:
```suggestion
- `showMeasurementUnit` props.
```

**Comment by klzns** [comment]
@regis-samurai can you provide a link to a workspace with that feature, please?

Also can you let us modify your pull request: https://help.github.com/en/github/collaborating- with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by regis-samurai** [comment]
> @regis-samurai can you provide a link to a workspace with that feature, please?
> 
> Also can you let us modify your pull request: https://help.github.com/en/github/collaborating- with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Actually, this checkbox already checked: 
Captura de Tela 2020-04-24 às 10 36 58

And the workspace running this feature is on the link: https://multiplier--cbyio.myvtex.com/

File: docs/README.md (hunk 3 above)
Comment (guerreirobeatriz) [inline docs/README.md:56]:
```suggestion
| `warningQuantityThreshold` | `number` | Displays the number of remaining items in stock if the total amount is less than or equal to the number configured in this prop. | `0` |
| `size` | `enum`| Preset values for `font-size` and `padding` to the component. Possible values are `small`, `regular` (medium size) and `large`. | `small` |
| `showLabel` | `boolean` | Defines whether a label is shown (`true`) or not (`false`). | `true` |
| `showMeasurementUnit` | `boolean` | Defines whether a measurement unit is shown (`true`) or not (`false`). | `false` |
```

File: docs/README.md [hunk 5]
```diff
@@ -53,6 +53,7 @@ Check an example of a Product Details Page built using Flex Layout with the `pro
...
 
 ### NumericSize
```
Comment (guerreirobeatriz) [inline docs/README.md:59]:
```suggestion
```

Comment (guerreirobeatriz) [reply docs/README.md:59]: This whole section (`NumericSize`) should be removed from documentation.

**Comment by klzns** [comment]
I updated the PR to show the measurementUnit if it's different from the default `un`

So there's no need for the prop anymore

https://breno--storecomponents.myvtex.com/orange-juice/p

**Comment by regis-samurai** [comment]
> I updated the PR to show the measurementUnit if it's different from the default `un`
> 
> So there's no need for the prop anymore
> 
> https://breno--storecomponents.myvtex.com/orange-juice/p

@klzns Perfect!

File: docs/README.md [hunk 6]
```diff
@@ -88,4 +88,4 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 <!-- prettier-ignore-end -->
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
-This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
\ No newline at end of file
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
```
Comment (guerreirobeatriz) [inline docs/README.md:91]:
```suggestion
This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind are welcome!
```

**Comment by klzns** [comment]
@allcontributors add @regis-samurai for code

**Pull request merged**